### PR TITLE
[TR-107] Added commands `kyc-deploy`, `kyc-manage`

### PR DESCRIPTION
**Pull request by voith**
TODO: 

- [ ] Add docs/tutorial( should include a link to create flattened solc output)

### CODE REVIEW

File: sto/cli/main.py [hunk 1]
```diff
@@ -601,6 +601,51 @@ def version(config: BoardCommmadConfiguration):
...
+    Deploys Kyc contract to desired ethereum network
+    network, ethereum-abi-file, ethereum-private-key, ethereum-node-url are required args
+    """
+    from sto.ethereum.utils import deploy_contract
+    deploy_contract(config, contract_name='BasicKYC')
+
+
+@cli.command(name="kyc-manage")
+@click.option('--whitelist-address', required=True, help="address to whitelist", type=str)
+@click.option('--nonce', required=False, help="Print only first N entries", default=0, type=int)
```
Comment (miohtama) [inline sto/cli/main.py]: Not sure if this is a correct option/comment here?

File: sto/cli/main.py [hunk 2]
```diff
@@ -601,6 +601,51 @@ def version(config: BoardCommmadConfiguration):
...
 
+@cli.command(name="kyc-deploy")
+@click.pass_obj
+def kyc_deploy(config: BoardCommmadConfiguration):
+    """
+    Deploys Kyc contract to desired ethereum network
+    network, ethereum-abi-file, ethereum-private-key, ethereum-node-url are required args
+    """
+    from sto.ethereum.utils import deploy_contract
+    deploy_contract(config, contract_name='BasicKYC')
```
Comment (miohtama) [inline sto/cli/main.py]: We can layer change this to deploy with you set `--transfer-mode=restricted` when doing an issuance.

File: sto/cli/main.py [hunk 3]
```diff
@@ -601,6 +601,51 @@ def version(config: BoardCommmadConfiguration):
     print(config.version)
 
 
+@cli.command(name="kyc-deploy")
+@click.pass_obj
+def kyc_deploy(config: BoardCommmadConfiguration):
```
Comment (miohtama) [inline sto/cli/main.py]: Cam you add a test case covering this?

File: sto/cli/main.py [hunk 4]
```diff
@@ -601,6 +601,51 @@ def version(config: BoardCommmadConfiguration):
...
+    """
+    from sto.ethereum.utils import deploy_contract
+    deploy_contract(config, contract_name='BasicKYC')
+
+
+@cli.command(name="kyc-manage")
+@click.option('--whitelist-address', required=True, help="address to whitelist", type=str)
+@click.option('--nonce', required=False, help="Print only first N entries", default=0, type=int)
+@click.pass_obj
+def kyc_manage(config: BoardCommmadConfiguration, whitelist_address, nonce):
```
Comment (miohtama) [inline sto/cli/main.py]: Also can you add a test case covering this?

**Comment by voith** [comment]
closing to check if change is reflected in #dev channel

**Comment by voith** [comment]
closing in favour of #3

**Pull request closed without merging**